crates/rpc/rpc/src/validation.rs

```diff
@@ -0,0 +1,93 @@
+use alloy_rpc_types_beacon::relay::{
+    BuilderBlockValidationRequest, BuilderBlockValidationRequestV2, BuilderBlockValidationRequestV3,
+};
+use async_trait::async_trait;
+use jsonrpsee::core::RpcResult;
+use reth_chainspec::ChainSpecProvider;
+use reth_provider::{
+    AccountReader, BlockReaderIdExt, HeaderProvider, StateProviderFactory, WithdrawalsProvider,
+};
+use reth_rpc_api::BlockSubmissionValidationApiServer;
+use std::sync::Arc;
+use tracing::warn;
+
+/// The type that implements the `validation` rpc namespace trait
+pub struct ValidationApi<Provider> {
+    inner: Arc<ValidationApiInner<Provider>>,
+}
+
+impl<Provider> ValidationApi<Provider>
+where
+    Provider: BlockReaderIdExt
+        + ChainSpecProvider
+        + StateProviderFactory
+        + HeaderProvider
+        + AccountReader
+        + WithdrawalsProvider
+        + Clone
+        + 'static,
+{
+    /// The provider that can interact with the chain.
+    pub fn provider(&self) -> Provider {
+        self.inner.provider.clone()
+    }
+
+    /// Create a new instance of the [`ValidationApi`]
+    pub fn new(provider: Provider) -> Self {
+        let inner = Arc::new(ValidationApiInner { provider });
+        Self { inner }
+    }
+}
+
+#[async_trait]
+impl<Provider> BlockSubmissionValidationApiServer for ValidationApi<Provider>
+where
+    Provider: BlockReaderIdExt
+        + ChainSpecProvider
+        + StateProviderFactory
+        + HeaderProvider
+        + AccountReader
+        + WithdrawalsProvider
+        + Clone
+        + 'static,
+{
+    async fn validate_builder_submission_v1(
+        &self,
+        _request: BuilderBlockValidationRequest,
+    ) -> RpcResult<()> {
+        todo!()
+    }
+
+    async fn validate_builder_submission_v2(
+        &self,
+        _request: BuilderBlockValidationRequestV2,
+    ) -> RpcResult<()> {
+        todo!()
```

Comment:
we can get this in incrementally but then these should return an "unimplemted" error and not panic

Reply:
also needs `make book` to update the new book docs

Reply:
ok addressed both in [81b4877](https://github.com/paradigmxyz/reth/pull/12061/commits/81b4877b7161f1d4a12a51120f89e12d73110cfe) and cleaned up the PR description so it's clear what the next steps will be.